Casks/m/moradownloader.rb

```diff
@@ -8,15 +8,13 @@
   homepage "https://mora.jp/"
 
   livecheck do
-    url "https://mora.jp/contents/data/system/noticeTouch.json"
-    regex(/Mac.*?v?\.?(\d+(?:\.\d+)+)/i)
+    url "https://downloader.mora.jp/mac/moradownloader.json"
+    regex(/(\d+(?:\.\d+)+)/i)
     strategy :json do |json, regex|
-      json["noticelist"]&.map do |notice|
-        match = notice["context"]&.match(regex)
-        next if match.blank?
+      match = json["versionInfo"]["latestVersionName"]&.match(regex)
```

Comment:
```suggestion
      match = json.dig("versionInfo", "latestVersionName")&.match(regex)
```

We use `#dig` when working with multi-level keys, as it will return `nil` if any of the provided keys are missing and that will be caught by subsequent guards like safe navigation (`&.`), etc.